ts/editor/NoteEditor.svelte

```diff
@@ -572,11 +538,14 @@ the AddCards dialog) should be implemented in the user of this component.
         </Absolute>
     {/if}
 
-    <!-- for init maskeditor -->
-    <div id="io-mask-editor" style="display: none;" />
+    {#if imageOcclusionMode && isImageOcclusion}
```

Comment:
Could we perhaps avoid the need for a separate isImageOcclusion boolean, and just check for imageOcclusionMode not being undefined?